com.chartboost.mediation/Editor/Adapters/AdaptersWindow.cs

```diff
@@ -168,10 +173,10 @@ private static void CreateAdapterTable(VisualElement root)
         
             var androidVersionLabel = new Label("Android");
             androidVersionLabel.name = "header-version";
-            adapterNameLabel.tooltip = "Android Version of Ad Adapters.";
+            androidVersionLabel.tooltip = "Android Version of Ad Adapters.";
             headers.Add(androidVersionLabel);
         
-            var iosVersionLabel = new Label("iOS");
+            var iosVersionLabel = new Label("IOS");
```

Comment:
nit: Should this not be iOS?
```suggestion
            var iosVersionLabel = new Label("iOS");
```

Reply:
I think it has something to do with displaying case-sensitive UI elements in Unity. But I could be wrong - it's been a while since I last heard this explanation.

Reply:
@cb-jpadilla @chauduyphanvu Let me clear this up, normally for any Unity serializable fields anything starting with lowercase will be automatically Uppercased. However, in this instance since we are talking UI, I could make it iOS to make it match the correct naming. I have updated all the instances referring to this. I am used to the first scenario, so I tend to be careless when handling this specific naming 😢